src/org/labkey/test/components/ui/ontology/OntologyTreePanel.java

```diff
@@ -71,6 +79,8 @@ protected ElementCache newElementCache()
     protected class ElementCache extends Component<?>.ElementCache
     {
         WebElement nodeContainer = Locator.tag("ul").parent().findWhenNeeded(this);
+        Locator activeNode = Locator.tagWithClass("div", "filetree-node-active");
```

Comment:
does this mean the finder for TreeNodeFinder.activeOnly() is looking for a different locator now?
To this point, I've been using TreeNodeFinder to build node locators